src/routes/route.tsx

```diff
@@ -41,6 +42,10 @@ export const Router = createBrowserRouter([
                 path: "home",
                 element: <PrivateRoute><Home /></PrivateRoute>
             },
+            {
+                path: "menu",
+                element: <PrivateRoute><Menu /></PrivateRoute>
```

Comment:
Seria mais interessante passar o <PrivateRoute> uma vez só, ao invés de passar envolta de toda Tag que for ser implementada